Bottom: d015f6ec028734c73e0849ae78907b6598975ec6
Top:    c7d39c932d4b299f353108f2416954322ff8e7bc
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-30 17:06:05 +0800

Refresh of autofs-5.0.7-fix-dumpmaps-multi-output.patch

---

diff --git a/lib/master.c b/lib/master.c
index efed3ba..2830f8d 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -1286,7 +1286,7 @@ static void print_map_info(struct map_source *source)
 	int argc = source->argc;
 	int i, multi, map_num;
 
-	multi = (source->type && strcmp(source->type, "multi"));
+	multi = (source->type && !strcmp(source->type, "multi"));
 	map_num = 1;
 	for (i = 0; i < argc; i++) {
 		if (source->argv[i] && *source->argv[i] != '-') {
@@ -1302,7 +1302,7 @@ static void print_map_info(struct map_source *source)
 		if (i >= argc)
 			return;
 
-		if (strcmp(source->argv[i], "--"))
+		if (!strcmp(source->argv[i], "--"))
 			continue;
 
 		if (source->argv[i]) {
